diff --git a/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec b/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec
index 248e1f4..d910608 100644
--- a/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec
+++ b/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.ios.deployment_target = "12.4"
+  s.ios.deployment_target = "12.0"
   s.osx.deployment_target = "10.15"
   s.tvos.deployment_target = "12.4"
   s.visionos.deployment_target = "1.0"
